_data/faq.yml

```diff
@@ -191,10 +191,12 @@
     - question: Can I try Gruntwork Software before committing to an annual contract?
       answer: |
         <p>
-          So, you just want a sneak peek? Before any contracts are signed, we'd be happy to do a walkthrough of our
-          code, training, and process. You can also try out parts of the IaC Library for free by checking out our
-          <a href="/infrastructure-as-code-library/#open-source" title="Free Open Source modules by Gruntwork">open
-          source modules</a>.
+          Here are three ways you can try out Gruntwork Software with no commitment:
+          <ol>
+            <li>You can try out parts of the IaC Library for free by checking out our <a href="/infrastructure-as-code-library/#open-source" title="Free Open Source modules by Gruntwork">open source modules</a>.</li>
+            <li>Before any contracts are signed, we'd be happy to do a walkthrough of our code, training, and process.</li>
+            <li>We offer a 30-day money-back guarantee, so if for any reason you're not happy after signing up, we'll refund 100% of your money in the first 30 days. Check out "What if I want to cancel my Gruntwork Subscription?" for more info.</li>
```

Comment:
Add link to the "What if I want to cancel" FAQ entry

Reply:
Done